tests/e3sm_diags/driver/utils/test_regrid.py

```diff
@@ -283,7 +283,7 @@ def test_regrids_to_first_dataset_with_equal_latitude_points(self, tool):
         else:
             assert_identical(result_b, expected_b)
 
-    @pytest.mark.parametrize("tool", ("xesmf", "regrid2"))
+    @pytest.mark.parametrize("tool", ("regrid2",))  # Skip xesmf for this test
```

Comment:
I'm curious why xesmf is removed from the tests? I noticed the test still passes with xesmf, which I would think we need to keep in the tests.

Also xCDAT ignores the `regrid_method` parameter when using `regrid2` since it is always `conservative`. We can raise a logger info message that the method will always be `conservative` and the `regrid_method` parameter is ignored. 

Or to make things cleaner, can even deprecate the option to use xCDAT's "regrid2" since we it is not used or tested in e3sm_diags v3.0.0.